spec/requests/users_spec.rb

```diff
@@ -1,17 +1,20 @@
 require 'rails_helper'
 
 RSpec.describe 'Users', type: :request do
-  describe 'GET /index' do
+  describe 'GET /index and render place holder page' do
     it 'returns http success' do
-      get '/users/index'
+      get '/users'
       expect(response).to have_http_status(:success)
+      expect(response.body.include?('List of users')).to be true
     end
   end
 
-  describe 'GET /show' do
+  describe 'GET /show and render place holder page' do
     it 'returns http success' do
-      get '/users/show'
+      @usr = User.create name: 'Testuser', photo: 'www.photo', bio: 'Test bio', posts_counter: 0
+      get "/users/#{@usr.id}"
       expect(response).to have_http_status(:success)
+      expect(response.body.include?('Given user with recent posts')).to be true
     end
   end
```

Comment:
- [ ] Spec file should include tests that ensure the correct view template is rendered.
Example: `expect(response).to render_template(:show)`, please kindly add for `show` and `index` as well 🙏🏾